receiver/jmxmetricsreceiver/README.md

```diff
@@ -0,0 +1,104 @@
+# JMX Metrics Receiver
+
+### Overview
+
+The JMX Metrics Receiver will work in conjunction with the upcoming OpenTelemetry JMX Metric Gatherer to
+report metrics from a target MBean server using your custom `otel` helper-utilizing Groovy script.
+
+Status: alpha
+
+### Details
+
+This receiver will launch a child JRE process running the JMX Metric Gatherer configured with your specified JMX
+connection information and target Groovy script.  It can report metrics to an existing otlp metric receiver specified
+by the `collector_endpoint` configuration option.
+
+# Configuration
+
+Note: this receiver is in alpha and functionality and configuration fields are subject to change.  It is currently
+partially implemented.
+
+Example configuration:
+
+```yaml
+receivers:
+  jmx_metrics:
+    service_url: service:jmx:rmi:///jndi/rmi://<my-jmx-host>:<my-jmx-port>/jmxrmi
+    groovy_script: /opt/my/groovy.script
```

Comment:
I think it's more user friendly to be able to put program text directly in yaml if desired.

We need to add a collector feature to be able to reference file paths but once that's solved any component will be able to use it. So I think putting the program text here is better.